fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,3 +1,12 @@
-This is a native Android user interface application for openHAB - an open source home automation system. The open Home Automation Bus (openHAB) project aims at providing a universal integration platform for all things around home automation. It is designed to be absolutely vendor-neutral as well as hardware/protocol-agnostic. openHAB brings together different bus systems, hardware devices and interface protocols. Supported technologies include KNX, Modbus, DMX, Philips Hue, PLCBus, Sonos, Plugwise, IHC/Elko, RFXCOM, Homematic, Koubachi, Pulseaudio, Samsung TV, Bluetooth, One-Wire, Novelan, Asterisk, generic serial devices and many others. It supports all openHAB features plus NFC tags based sitemap navigation and item control, automatic discovery of your openHAB and remote connection back to your house over the Internet with my.openHAB service. openHAB is the integration and control point of your Smart Home. You will find openHAB at http://www.openhab.org/
+This is a native Android user interface application for openHAB - an open source home automation system
 
-Please note: GCM and crash analytics has been removed from the F-Droid version. Unlike in other apps it is built directly from upstream.
+The open Home Automation Bus (openHAB) project aims at providing a universal integration platform for all things around home automation. It is designed to be absolutely vendor-neutral as well as hardware/protocol-agnostic. openHAB brings together different bus systems, hardware devices and interface protocols. Supported technologies include KNX, Modbus, DMX, Philips Hue, PLCBus, Sonos, Plugwise, IHC/Elko, RFXCOM, Homematic, Koubachi, Pulseaudio, Samsung TV, Bluetooth, One-Wire, Novelan, Asterisk, generic serial devices and many others. It supports all openHAB features plus NFC tags based sitemap navigation and item control, automatic discovery of your openHAB and remote connection back to your house over the Internet with my.openHAB service. openHAB is the integration and control point of your Smart Home. You will find openHAB at http://www.openhab.org/
+
+<b>Important note</b><br>
```

Comment:
A user from Portugal asks in the reviews how this app works, I think he has no clue that he needs a oh server.